### PR TITLE
Troubleshooting update

### PR DESCRIPTION
**Pull request by weshayutin**
* Update the stale trouble shooting section of the docs here
* Perhaps collab with Andy to make this the source of future prod updates.
  * need combine the two docs into one and then move forward.

### CODE REVIEW

File: docs/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -1,91 +1,108 @@
...
+    ```
+    oc logs -f deployment.apps/volume-snapshot-mover -n openshift-adp
+    ```
+    
+2. Velero commands
+    1. Alias the velero command: 
+    ```
+    alias velero='oc -n openshift-adp exec deployment/velero -c velero -it -- ./velero'
+    ```
+    2. Get the backup details:
```
Comment (sseago) [inline docs/TROUBLESHOOTING.md:89]: s/backup/restore/

File: docs/TROUBLESHOOTING.md [hunk 2]
```diff
@@ -1,91 +1,108 @@
...
+    ```
+    alias velero='oc -n openshift-adp exec deployment/velero -c velero -it -- ./velero'
+    ```
+    2. Get the backup details: 
+    ```
+    velero restore describe <restoreName> --details
+    ```
+    3. Get the backup logs: 
+    ```
+    velero backup logs <restoreName>
```
Comment (sseago) [inline docs/TROUBLESHOOTING.md:95]: s/backup/restore/

File: docs/TROUBLESHOOTING.md [hunk 3]
```diff
@@ -139,12 +160,23 @@ If the issue still persists, [create a new issue](https://github.com/openshift/o
...
   or restore was interrupted. If this is the case, it cannot resume. 
 
-  - For further details on your backup, run the command `velero backup describe <backup-name>`.
-  And for more details on your restore, run `velero restore describe <backup-name>`.
-
-  - You can delete the backup with the command `oc delete backup <backup-name> -n openshift-adp`,
-  and delete the restore with the command `oc delete restore <restore-name> -n openshift-adp`
+  - For further details on your backup, run the command:
+  ```
+  velero backup describe <backup-name>
```
Comment (sseago) [inline docs/TROUBLESHOOTING.md:165]: Optionally, include `--details` if you want to see the resource list or volume-level information.

File: docs/TROUBLESHOOTING.md (hunk 1 above)
Comment (weshayutin) [reply docs/TROUBLESHOOTING.md:89]: doh.. thanks scott!

File: docs/TROUBLESHOOTING.md (hunk 3 above)
Comment (weshayutin) [reply docs/TROUBLESHOOTING.md:165]: that one is on the next line

Comment (weshayutin) [reply docs/TROUBLESHOOTING.md:165]: oh.. adding..

File: docs/TROUBLESHOOTING.md [hunk 4]
```diff
@@ -1,91 +1,108 @@
...
-  `oc logs -l name=restic`. Check for errors.  
+This section includes steps to debug a failed backup. For more specific issues related to Restic/CSI/volume snapshots check out the following section. Both the OpenShift commands and native Velero commands are useful in troubleshooting.
+1. OpenShift commands
+    1. Check for validation errors in the backup by running the following command,
+    ```
+    oc describe backup <backupName> -n openshift-adp
+    ```
+    2. Check the Velero logs
+    ```
+    oc logs -f deployment.apps/velero -n openshift-adp
```
Comment (kaovilai) [inline docs/TROUBLESHOOTING.md:33]:
```suggestion
    oc logs -f deploy/velero -n openshift-adp
```
could be shortened.

**Review by sseago — APPROVED** [review]

**Pull request merged**